web/src/views/DandisetLandingView/ExternalDandisetServicesDialog.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <v-menu
+    offset-y
+    left
+  >
+    <template
+      #activator="{ on, attrs }"
+    >
+      <v-btn
+        id="external-dandiset-services"
+        outlined
+        block
+        v-bind="attrs"
+        v-on="on"
+      >
+        <v-icon
+          color="primary"
+          left
+        >
+          mdi-web
+        </v-icon>
+        <span>Open with</span>
```

Comment:
```suggestion
        <span>Visualize</span>
```

I like how NEMAR and OpenNeuro use verbs that describe what can be done by clicking the button (e.g. compute, analyze).  As other external services are added, we can add other general categories.

NEMAR:
<img width="598" alt="image" src="https://github.com/user-attachments/assets/c4de0998-c839-47b5-9fee-1359127cd1fe">

OpenNeuro:
<img width="953" alt="image" src="https://github.com/user-attachments/assets/ebb800a9-4b80-4001-9e3f-08f4c2db8236">

Reply:
I think in this case "Open with" is more accurate than "Visualize" because in Neurosift you can browse, explore and (with Dendro) analyze. Visualize is more relevant when opening an individual NWB file. Really, for the Dandiset view, Neurosift is an alternate way of viewing a subset of the information on the DANDI site, with the ability to do Neurosift-specific exploration and analysis.

Reply:
And "Open with" is what I suggested based on what we have in file browser -- making it consistent is good. 
I do see a good UX in clear verbs, but indeed here it is not applicable IMHO, **but** it brings an idea to annotate those external services with "actions" or alike, which would be `['visualize', 'analyze']` or `['download']` which we could then somehow depict (icons?) along with the entries within `Open with` drop down.

Reply:
That sounds good, thanks